packages/vuetify/src/components/VCheckbox/VSimpleCheckbox.ts

```diff
@@ -80,7 +80,7 @@ export default Vue.extend({
 
     return h('div', {
       ...data,
```

Comment:
Is there a reason we aren't just using mergeData for the whole lot?

Reply:
I think we can do it. It is not clear why it was not done so initially, therefore, I presented such a solution. Should I remake it into what you proposed?

Reply:
@jacekkarczmarczyk did it in #11358, maybe he remembers

Reply:
Becase in that PR there were only listeners to merge, if there are more thing to merge `mergeData` can be used as well